### ISSUE TITLE
KeyError: 'customData'

### ISSUE THREAD

**Issue by Gabriele007xx** [Richard-Weiss/Bing-Creator-Image-Downloader#6]
Ok so i have tried to download again.
First i downloaded and unzipped the latest release. Then, seeing the new requirements, i downloaded them too.
After that i took mu _U cookie following your guide on your reddit comment. I pasted it in the .evi file, right after the equal character.
I deleted the collection property as i want to download all of my collections (The AI one made up from 1500 images and another with just one), including the quotation marks.
I renamed the file in '.evi'. 
I run the script.
Did i miss anything?
Sorry to bother you again btw 😅
Also, you should delete the image_clipboard.txt file since it's not used anymore with this method.
![image](https://github.com/Richard-Weiss/Bing-Creator-Image-Downloader/assets/97365913/74acfd83-6e5c-4ca7-8d16-595d17ba846a)

**Comment by Richard-Weiss** [comment]
@Gabriele007xx I'll take a look trying to replicate it.

**Comment by Richard-Weiss** [comment]
Could you also include the content of your .env file with the content of the COOKIE omitted?

**Comment by Richard-Weiss** [comment]
@Gabriele007xx Can you try pasting this code after line 51?:
```
with open('collection_dict_dump.txt', 'w') as f:
    f.write(json.dumps(collection_dict))
```
So the section looks like this:
```
if len(collection_dict) == 0:
    raise Exception('No collections were found for the given cookie.')
with open('collection_dict_dump.txt', 'w') as f:
    f.write(json.dumps(collection_dict))
image_tuples = []
```
After that please share the content of the `collection_dict_dump.txt` file so I can analyze it.  
I'm missing the required data to replicate it on my end.

**Comment by Gabriele007xx** [comment]
Pasted like you said. 
[Uploading collection_dict_dump.txt…]()

**Comment by Gabriele007xx** [comment]
Pasted like you said
[collection_dict_dump.txt](https://github.com/Richard-Weiss/Bing-Creator-Image-Downloader/files/13231682/collection_dict_dump.txt)

**Comment by Richard-Weiss** [comment]
Hey @Gabriele007xx,
I'll take a look at it tomorrow or on the weekend.

**Comment by Richard-Weiss** [comment]
@Gabriele007xx should be fixed with commit [1944893](https://github.com/Richard-Weiss/Bing-Creator-Image-Downloader/commit/194489312064d1118ee73def61777dbbe0b89772)

**Comment by Richard-Weiss** [comment]
@Gabriele007xx I've added another commit to improve validation further. Be sure to pull it before trying it again.
